src/lib/components/Creatibutors/CreatibutorsModal.js

```diff
@@ -210,28 +248,20 @@ export class CreatibutorsModal extends Component {
         }
       });
 
+      let idString =[];
+      creatibutor.identifiers.forEach((i, idx) => {
+        idString.push(this.makeIdEntry(i));
+      });
+
       return {
         text: creatibutor.name,
-        value: orcid.identifier,
+        value: creatibutor.id,
         extra: creatibutor,
         key: creatibutor.id,
         content: (
           <Header>
             <Header.Content>
-              {creatibutor.name} (
-              <Image
-                src="/static/images/orcid.svg"
-                className="inline-id-icon"
-                verticalAlign="middle"
-              />
-              {orcid.identifier})
-              <a
-                href={`https://orcid.org/${orcid.identifier}`}
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                <Icon link name="external alternate" className="spaced-left" />
-              </a>
+              {creatibutor.name} ({idString})
```

Comment:
```suggestion
              {creatibutor.name} ({idString})
```

Forgot something. I don't think it's common, but in case `idString` is empty (i.e. there's no `identifiers` key in the metadata, or it's an empty list) the parentheses after the name shouldn't be displayed at all also.

Reply:
fixed with thanks for the assistance.